packages/adapter-static/index.js

```diff
@@ -11,6 +11,9 @@ export default function ({ pages = 'build', assets = pages, fallback } = {}) {
 		name: '@sveltejs/adapter-static',
 
 		async adapt({ utils }) {
+			if (utils.update_ignores({ patterns: [pages, assets] })) {
+				utils.log.minor('Ignore files updated');
```

Comment:
why not just move the updated/not-updated logic + logging into the `update_ignores` function, and then we can drop the repeated `if` statement in each adapter

Reply:
I just wasn't sure if all the consumers of this function would desire that to happen.